### PR TITLE
log cleanups

### PR DESCRIPTION
**Pull request by michaelkirk**
Sorry, a couple things I hadn't pushed up for #320 yet.

The workflow I'm used to is to respond to PR feedback line by line as I work on it, then re-tag "Please take another look..." for review when I'm finished. This is potentially confusing, so in the future I'll try to address everything before making confusing statements like "done". 😓 

Also took the opportunity to shave off a little more log spam.

### CODE REVIEW

File: widgetry/src/lib.rs [hunk 1]
```diff
@@ -24,6 +24,9 @@
 
 //#![warn(missing_docs)]
 
+#[macro_use]
+extern crate log;
```
Comment (michaelkirk) [inline widgetry/src/lib.rs:28]: My plan is to eventually do this everywhere, and gradually convert println's to log methods. LMK if you'd like to discuss more before I keep doing this.

Comment (dabreegster) [reply widgetry/src/lib.rs:28]: SGTM for the random `println`s. If you want to modify any of the `timer.note` calls, let's discuss a plan first.

**Comment by dabreegster** [comment]
Woops, I'm still used to Google's Critique UI, which batches replies to comments. I'll wait for the "PTAL" next time. This one LGTM, confirming ready?

**Comment by michaelkirk** [comment]
> Woops, I'm still used to Google's Critique UI, which batches replies to comments.

heh, I guess GH has something like this now with the "start a review" process, but for whatever reason it's not always quite at hand, and... old dog/new trick, etc.

> This one LGTM, confirming ready?

Yep!

**Pull request merged**